ktlint/src/main/kotlin/com/pinterest/ktlint/internal/PrintASTSubCommand.kt

```diff
@@ -77,6 +77,6 @@ internal class PrintASTSubCommand : Runnable {
 
     companion object {
         internal const val COMMAND_NAME = "printAST"
-        private const val STDIN_FILE = "<stdin>"
+        private const val STDIN_FILE = "<text>"
```

Comment:
the code explicitly uses this as a flag for stdin. we should probably move this into a constant somewhere.

Reply:
Sorry, missed that as it is not a constant. Yes, you are right - would be better to add `const val STDIN_FILE_NAME = "<stdin>"` to https://github.com/pinterest/ktlint/blob/master/ktlint-core/src/main/kotlin/com/pinterest/ktlint/core/KtLint.kt and use it here and in other places, so in the future it will not happen again.

Generally, I think, we should bring some integration test to command line module.

Reply:
@Tapchicoma I'll make that fix in a separate PR.

Reply:
@Tapchicoma updated with the constant